internal/vcs/git/diff_search.go

```diff
@@ -168,6 +168,20 @@ func RawLogDiffSearch(ctx context.Context, repo gitserver.Repo, opt RawLogDiffSe
 		return nil, false, fmt.Errorf("invalid options: Query.IsCaseSensitive != Paths.IsCaseSensitive")
 	}
 
+	// We do a search with git log returning just the commits (and source sha).
+	onelineCommits, complete, err := rawLogSearch(ctx, repo, opt)
+	if err != nil {
+		return nil, complete, err
+	}
+
+	// We then search each commit to further filter the results.
+	results, complete2, err := rawShowSearch(ctx, repo, opt, onelineCommits)
```

Comment:
```suggestion
	onelineCommits, completeLog, err := rawLogSearch(ctx, repo, opt)
	if err != nil {
		return nil, complete, err
	}

	// We then search each commit to further filter the results.
	results, completeShow, err := rawShowSearch(ctx, repo, opt, onelineCommits)
```

or at least `completeShow` rather than `complete2`

Reply:
I agree, that is a better name :) I chose this name since it is consistent with the older code that I was transforming.